docs/Tutorials/CCE.md

```diff
@@ -292,6 +292,9 @@ Here are some notes about the different options in the YAML input file:
   simulation that provided the boundary dataset.
 - `FixSpecNormalization` should always be `False` unless you are using a
   particualy old version of SpEC
+- `DescendingM` should always be `False`. This option is to maintain
+  compatibility with older versions of SpEC. Our
```

Comment:
Current SpEC still dumps metric modal data with descending m (even though this option has been disabled in a default run). I would be more explicit here:

`This option is to support metric modal data from SpEC.`